src/main/java/hudson/plugins/jira/CredentialsHelper.java

```diff
@@ -142,10 +142,10 @@ protected static FormValidation doCheckFillCredentialsId(
 
 	protected static Optional<StandardUsernamePasswordCredentials> findCredentials(
 		Item item, String credentialsId, String uri) {
-		if (isNullOrEmpty(credentialsId)) {
-			return absent();
+		if (StringUtils.isEmpty(credentialsId)) {
+			return null;
```

Comment:
don't think we should return `null` when using Optionals

Reply:
agree do not return null if using Optional